backend/backend/services/call_whitelabel.py

```diff
@@ -0,0 +1,99 @@
+import logging
+from typing import Literal
+
+from tenacity import after_log, retry, retry_if_exception_type, stop_after_delay, wait_random_exponential
+
+import whitelabel_client
+from backend.config import get_settings
+from backend.const import WHITELABEL_ORDER_STATUS_MAPPING
+from backend.db import get_session
+from backend.models.tables import Message, Order, UserType
+from whitelabel_client import ApiException, MessageWrite, MessageWriteAuthor, ProjectItemUpdate
+
+logger = logging.getLogger(__name__)
+
+configuration = whitelabel_client.Configuration(
+    host=get_settings().whitelabel_endpoint,
+    api_key={"authentication_token": get_settings().whitelabel_client_key},
+)
+
+
+def post_message(message_id: int, send_as: Literal[UserType.PROVIDER_MANAGER, UserType.COORDINATOR]) -> None:
+    with get_session() as session:
+        message = session.get(Message, message_id)
+        if not message:
+            raise ValueError(f"Message with id {message_id} not found")
+
+        message.synced = False
+        session.commit()
+
+        _post_message(message, send_as)
+
+        message.synced = True
+        session.commit()
+
+
+@retry(
+    reraise=True,
+    retry=retry_if_exception_type(ApiException),
+    wait=wait_random_exponential(multiplier=1, max=60),
+    stop=stop_after_delay(max_delay=get_settings().whitelabel_max_retry_delay_s),
```

Comment:
get_settings() is called at decoration, which can cause potential problems if settings change or are loaded lazily.
Consider:
`MAX_RETRY_DELAY = get_settings().whitelabel_max_retry_delay_s`